packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -1486,7 +1486,7 @@ description: Dart SDK extensions for dart:ui
 homepage: http://flutter.io
 # sky_engine requires sdk_ext support in the analyzer which was added in 1.11.x
 environment:
-  sdk: '>=1.11.0 <3.0.0'
```

Comment:
I'm guessing this in an opportunistic change. Want to do this in a separate commit in case this gets rolled back, and to not confuse a stable cherry pick attempt?

Reply:
Oops, I did not mean to commit this - but I was originally using this branch for something else 😓